RISCVGenSearchableTables.inc(20342): warning C4804: '<': unsafe use of type 'bool' in operation

MSVC builds are reporting:
```
E:\llvm\ninja\lib\Target\RISCV\RISCVGenSearchableTables.inc(20342): warning C4804: '<': unsafe use of type 'bool' in operation
E:\llvm\ninja\lib\Target\RISCV\RISCVGenSearchableTables.inc(20344): warning C4804: '>': unsafe use of type 'bool' in operation
```

--- a/llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td
+++ b/llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td
@@ -564,7 +564,7 @@ class RISCVVPseudo<dag outs, dag ins, list<dag> pattern = [],
   Instruction BaseInstr = !cast<Instruction>(PseudoToVInst<NAME>.VInst);
   // SEW = 0 is used to denote that the Pseudo is not SEW specific (or unknown).
   bits<8> SEW = 0;
-  bit IsAltFmt = !eq(AltFmtType.Value, IS_ALTFMT.Value);
+  bits<1> IsAltFmt = !eq(AltFmtType.Value, IS_ALTFMT.Value);
   bit IncludeInInversePseudoTable = 1;
 
   let UseNamedOperandTable = true;

